src/languages.js

```diff
@@ -30,6 +30,10 @@ Languages.get = function(code, key, callback) {
 	var languageData;
 
 	fs.readFile(path.join(__dirname, '../public/language/', code, key), { encoding: 'utf-8' }, function(err, data) {
+		if (err) {
```

Comment:
How do you know that the error always indicates that a file is missing? Also if an error exists I would assume that messageData is null or undefined, so accessing the parsedMessage property would fail.